File: app/src/main/java/com/chiron/playpalyer/media/IDecoderStateListener.java
package com.chiron.playpalyer.media;

public interface IDecoderStateListener {
    void decoderPrepare(BaseDecoder decodeJob);
    void decoderReady(BaseDecoder decodeJob);
    void decoderRunning(BaseDecoder decodeJob);
    void decoderPause(BaseDecoder decodeJob);
    void decodeOneFrame(BaseDecoder decodeJob);
    void decodeFinish(BaseDecoder decodeJob);
    void decodeDestroy(BaseDecoder decodeJob);
    void decodeError(BaseDecoder decodeJob,String msg);
}

File: app/src/main/java/com/chiron/playpalyer/mediac/IDecoderStateListener.kt
package com.chiron.playpalyer.mediac

interface IDecoderStateListener {
    fun decoderPrepare(decodeJob:BaseDecoder?)
    fun decoderReady(decodeJob:BaseDecoder?)
    fun decoderRunning(decodeJob:BaseDecoder?)
    fun decoderPause(decodeJob:BaseDecoder?)
    fun decodeOneFrame(decodeJob:BaseDecoder?)
    fun decodeFinish(decodeJob:BaseDecoder?)
    fun decodeDestroy(decodeJob:BaseDecoder?)
    fun decodeError(decodeJob:BaseDecoder?,msg:String)
}